src/content/docs/tools/docker-extension.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Docker Extension
+description: Learn how to use the Daytona Docker extension to create, open, list, and delete Workspaces.
+---
+
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Self-Managed**
+</Label>
+
+Daytona provides a Docker extension for using an embedded version of the Daytona client and server tools directly within the Docker Desktop.
+
+The Docker extension allows you to create, open, list, and delete [Workspaces](/docs/usage/workspaces).
+
+View the Daytona Docker extension in the [Docker Desktop Extensions Marketplace](https://open.docker.com/extensions/marketplace?extensionId=daytonaio/docker-extension&tag=0.44.0) or [GitHub repository](https://open.docker.com/extensions/marketplace?extensionId=daytonaio/docker-extension&tag=0.44.0).
```

Comment:
Since its `Docker Desktop Extension Marketplace`, myb we should refer to it as `Docker Extension` instead of `Docker extension`